sample/Getting started.jl

```diff
@@ -13,6 +13,9 @@ macro bind(def, element)
     end
 end
 
+# ╔═╡ 9b62f2c6-8037-486f-b8ec-a95b577b8d74
+using CommonMark
+
 # ╔═╡ b129ba7c-953a-11ea-3379-17adae34924c
 md"# _Welcome to Pluto!_
```

Comment:
Should these be `cm"` here if this is meant to exercise CM.jl rather than MD.jl? Or have I missed some magic elsewhere?

Reply:
You're entirely right, thanks. I've fixed this.